package.json

```diff
@@ -66,7 +66,7 @@
     "eslint-plugin-ft-flow": "^3.0.1",
     "eslint-plugin-prettier": "^4.0.0",
     "jest": "^28.1.1",
-    "metro-react-native-babel-preset": "0.76.8",
+    "metro-react-native-babel-preset": "^0.77.0",
```

Comment:
let's make sure these dev deps are necessary and if true, make them the same versions as in example app

Reply:
https://github.com/callstack/react-native-pager-view/pull/834/commits/0a7534637f52e7d557be7852381de77514990f79